Nuro/Components/Sidebar/TabBarItem.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  TabBarItem.swift
+//  Nuro
+//
+//  Created by Samuel Dennis on 06/10/22.
+//
+
+import UIKit
+
+enum TabBarItem: Int, CaseIterable {
+    case parentTodayActivity, parentRoutine, parentActivityList, childRoutine
+    
+    func title() -> String {
+        switch self {
+        case .parentTodayActivity:
+            return "Aktivitas hari ini"
+        case .parentRoutine:
+            return "Rutinitas"
+        case .parentActivityList:
+            return "Daftar aktivitas"
+        case .childRoutine:
+            return "Rutinitas anak"
```

Comment:
Put in `Strings` file to make the title reusable on the sidebar and navigation bar title